plugins/org.eclipse.n4js/src/org/eclipse/n4js/external/NpmManager.java

```diff
@@ -174,7 +174,7 @@ public IStatus installDependencies(final Map<String, String> versionedNPMs, fina
 	 * @param monitor
 	 *            the monitor for the blocking install process.
 	 * @param triggerCleanbuild
-	 *            iff true, a clean build is triggered on all affected workspace projects.
```

Comment:
are you aware that: "iff" = "if and only if". In this case here "iff" seems more precise.

Reply:
yes but down the code there is `if (Platform.isRunning())`, then there is dispatch to sub type implementation where one implementation does nothing, while other one can exit early in certain conditions.

So.. it is actually less `iff` and more `iffy`.

Reply:
👍